src/cbc_sdk/audit_remediation/differential.py

```diff
@@ -0,0 +1,293 @@
+#!/usr/bin/env python3
+
+# *******************************************************
+# Copyright (c) VMware, Inc. 2020-2022. All Rights Reserved.
+# SPDX-License-Identifier: MIT
+# *******************************************************
+# *
+# * DISCLAIMER. THIS PROGRAM IS PROVIDED TO YOU "AS IS" WITHOUT
+# * WARRANTIES OR CONDITIONS OF ANY KIND, WHETHER ORAL OR WRITTEN,
+# * EXPRESS OR IMPLIED. THE AUTHOR SPECIFICALLY DISCLAIMS ANY IMPLIED
+# * WARRANTIES OR CONDITIONS OF MERCHANTABILITY, SATISFACTORY QUALITY,
+# * NON-INFRINGEMENT AND FITNESS FOR A PARTICULAR PURPOSE.
+
+"""Model and Query Classes for Differential Analysis"""
+
+from cbc_sdk.base import NewBaseModel, IterableQueryMixin, BaseQuery, CriteriaBuilderSupportMixin
+from cbc_sdk.platform import Job
+from cbc_sdk.errors import ApiError
+
+
+# Rate limits per 5 mins on an org basis
+SYNC_RATE_LIMIT = 350
+ASYNC_RATE_LIMIT = 100
+
+
+"""Differential Analysis Models"""
+
+
+class Differential(NewBaseModel):
+    """
+    Represents a Differential Analysis run.
+
+    Example:
+        >>> run = cb.select(Differential).newer_run_id(newer_run_id)
+        >>> print(*run)
+        >>> print(resp[0]._info)
```

Comment:
Our examples should not use _info instead we should show how to get one of the properties like `diff_results`

```
print(resp[0])
print(resp[0].diff_results)
```